coral/src/app/features/user-information/tenant-information/TenantInformationData.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { cleanup, render } from "@testing-library/react";
+import { TenantInformationData } from "src/app/features/user-information/tenant-information/TenantInformationData";
+
+const props = {
+  tenantName: "default",
+  contactPerson: "Klaw Administrator",
+  orgName: "Default Organization",
+  description: "This is the default description",
+};
+
+describe("TenantInformationData.tsx", () => {
+  afterAll(cleanup);
+
+  describe("renders correct DOM with provided props", () => {
+    it("renders correct DOM with required props", () => {
+      render(<TenantInformationData {...props} />);
+
+      expect(screen).toMatchSnapshot();
```

Comment:
*chichi* didn't we have the "snapshot tests are worthless" thing yesterday? 😅 

(Just a comment about this timing coincident - I think this right here is a good use case for snapshots!)

Reply:
Yes, the irony is not lost on me haha